packages/nx/src/command-line/release/config/conventional-commits.ts

```diff
@@ -0,0 +1,98 @@
+import { NxReleaseConfig } from './config';
+
+export const DEFAULT_CONVENTIONAL_COMMITS_CONFIG: NxReleaseConfig['conventionalCommits'] =
+  {
+    types: {
+      feat: {
+        semverBump: 'minor',
+        changelog: {
+          title: '🚀 Features',
+          hidden: false,
+        },
+      },
+      fix: {
+        semverBump: 'patch',
+        changelog: {
+          title: '🩹 Fixes',
+          hidden: false,
+        },
+      },
+      perf: {
+        semverBump: 'none',
```

Comment:
Patch right? How would users benefit from the performance improvements otherwise?

Reply:
This is consistent with existing functionality. `perf` types weren't triggering version bumps, but showed up in changelogs - https://github.com/nrwl/nx/blob/46592d79128b14ca3e98d62fe78be6aed4633505/packages/nx/src/command-line/release/utils/resolve-semver-specifier.ts#L10

I could default this to patch if you'd like, but it would be a breaking change.